collections/corine-land-cover-accounting-layers.yaml

```diff
@@ -0,0 +1,94 @@
+ The CLC inventory consists of 44 land cover and land use classes derived from a series of satellite missions since it was first established. 
+ There are altogether 5 mapping inventories implemented since 1986, producing five status layers (CLC1990, CLC2000, CLC2006, CLC2012, CLC2018), 4 CLC-Change layers for the corresponding periods (1990-2000, 2000-2006, 2006-2012, 2012-2018) and 4 CLC Accounting layers for 2000, 2006, 2012 and 2018. 
+ More information can be found in Annex 5 of the [Product User manual](https://land.copernicus.eu/user-corner/technical-library/clc-product-user-manual). 
+AdditionalInfoExternal:
+    Title: Additional info 
+    Path: corine-land-cover-accounting-layers/README.MD 
+Image: corine-land-cover-accounting-layers/thumbnail.png   
+Resolution: 100m
+GeographicalCoverage: EEA39 region
+TemporalAvailability: 2000, 2006, 2012, 2018
+UpdateFrequency: Every 6 years. 
+BandInformation: 
+  Table:
+      Columns:
+        - Name: Name
+          Title: Name
+        - Name: Description
+          Title: Description        
+      Rows:
+        - Name: CLC_ACC
+          Description: Corine Land Cover Accounting Layer 
+Contact: <a href="mailto:sdi@eea.europa.eu">European Environment Agency</a> 
+Provider: "[European Environment Agency](https://www.eea.europa.eu/)"
+ManagedBy: "[Sentinel Hub](https://www.sentinel-hub.com/)"
+Tags:
+  - accounting
+  - land cover
+  - CLMS
+  - corine
+  - derived data
+  - open data
+License: EEA standard re-use policy ([read more](https://www.eea.europa.eu/legal/copyright)).
+LicenseType: proprietary 
+LicenseUrl: https://www.eea.europa.eu/legal/copyright
+Resources:
+  - Group: Sentinel Hub Resources
+    EndPoint: creodias.sentinel-hub.com
+    Name: Sentinel Hub
+    Role: processor
+    Type: byoc-4c5441a6-6040-4dc4-a392-c1317bbd1031
+    CollectionId: 4c5441a6-6040-4dc4-a392-c1317bbd1031
+    Primary: true
+  - Group: xcube Resources
+    DatasetName: CUSTOM
+    CollectionId: 4c5441a6-6040-4dc4-a392-c1317bbd1031
+Configurations: 
+  - layer_name: Corine Land Cover Accounting Layer
+    evalscript_url: "https://custom-scripts.sentinel-hub.com/custom-scripts/copernicus_services/corine_land_cover_accounting_layer/script.js"
+    mosaicking_order: mostRecent
+    upsampling: BICUBIC
+    href: https://docs.sentinel-hub.com/api/latest/api/process/
+    rel: processing-expression
+Extent:
+  spatial:
+    bbox:
+      -
+        - -56
+        - 24
+        - 72
+        - 72
+  temporal:
+    interval:
+      -
+        - '2000-01-01T00:00:00Z'
+        - '2018-01-01T00:00:00Z'
+CubeDimensions:
+  x:
+    type: spatial
+    axis: x
+    extent:
+      - -56
+      - 72
+  y:
+    type: spatial
+    axis: y
+    extent:
+      - 24
+      - 72
+  t:
+    type: temporal
+    extent:
+      - '2000-01-01T00:00:00Z'
+      - '2018-01-01T00:00:00Z'
```

Comment:
There only seems to be data from `2018-01-01T12:20:00.000Z` onwards. That's a pretty weird cutoff, what's up with that?